config/backup.php

```diff
@@ -65,6 +65,9 @@
                 'local',
             ],
         ],
+
+        'temp_directory' => storage_path('app/backup-temp'),
```

Comment:
Rename this to `temporary_directory`. Also add a sentence explaining the option.